api/src/gameState/gameState.controller.ts

```diff
@@ -29,34 +29,46 @@ export class GameStateController {
   ) {}
 
   @Post('')
-  async createGame(): Promise<GameState> {
-    const res = await this.gameStateService.createGame();
-    return this.gameStateService.deserializer(res);
+  async createGame(
+    @Body('playerId', ParseIntPipe) playerId: number,
+  ): Promise<GameState> {
+    const res = await this.gameStateService.createGame(playerId);
+    return this.gameStateService.deserializerGameState(res);
+  }
+
+  @Get('')
+  async getGames(): Promise<{ data: Game[] }> {
+    try {
+      const result = await this.gameStateService.getGames({});
+      result.data.map((game) => this.gameStateService.deserializerGame(game));
+      return result;
+    } catch (error) {
+      throw new NotFoundException('No games was found');
```

Comment:
When getting the list of all games, if an error occurs it cannot be a not found error!